lib/gcpclient.js

```diff
@@ -38,7 +47,8 @@ GCPClient.prototype.authorize = function() {
  */
 GCPClient.prototype.showDeployDetail = function(userData, deployId) {
   var client = this;
-  return this.authorize().then(function(tokens) {
+  this.authorize().then(function(auth) {
```

Comment:
:( Looking into it I don't think I'm handling that at all. I THINK it will get new tokens but I'm not saving them so presumably that will not work indefinitely. Will spend some time testing that out.